gradle/libs.versions.toml

```diff
@@ -21,7 +21,6 @@ lombok = '1.18.34'
 odd-oddrn-generator = '0.1.17'
 odd-oddrn-client = '0.1.41'
 cel = '0.3.0'
-junit = '5.11.2'
 mockito = '5.16.0'
```

Comment:
JUnit 5, in particular, was causing a direct conflict that might be resolved by upgrading or aligning versions. However, since it’s brought in transitively by  one our direct test dependencies, maintaining version alignment doesn't seem worth pursuing.

Mockit and assertj are also included transitively and technically falls into the same category, but it didn’t cause any issues during this upgrade.

Overall, reducing the number of explicitly managed dependencies isn’t a bad idea, so I’m open to removing this and others as well.


@Haarolean What do you think?

Reply:
I don't think it's a good idea to use some random junit/mockito versions pulled transitively. Bumping some other unrelated dependency then can break things, and junit is kinda known for breaking things sometimes.

Reply:
Do you suggest that we continue to sync them? Because what motivated this change is to align them with what Spring boot Uses as it was failing due to a conflict

Reply:
I'd align them (explicitly) with whatever spring boot uses itself, so in this case we'd have the same issues like we have now (version mismatch) rather than suddenly failing tests (say, when spring boot is bumped next time). What do you think?

Reply:
Thanks for the suggestion. I reverted these changes